### PR TITLE
feat(graphql): Add alert below request when there were errors

### PR DESCRIPTION
**Pull request by malwilley**
This adds context as to why a line has been highlighted and summarizes errors in the response.

CleanShot 2023-06-29 at 15 20 19

### CODE REVIEW

**Comment by marandaneto** [comment]
Awesome, so in this case the user does not need to read the response payload at all most likely, thank you @malwilley

File: static/app/components/events/interfaces/request/graphQlRequestBody.tsx [hunk 1]
```diff
@@ -41,6 +47,32 @@ function getErrorLineNumbers(errors: GraphQlErrors): number[] {
...
+  }
+
+  return (
+    <StyledAlert
+      type="error"
+      showIcon
+      expand={
+        <List symbol="bullet">
+          {errors.map((error, i) => (
+            <li key={i}>{error.message}</li>
```
Comment (marandaneto) [inline static/app/components/events/interfaces/request/graphQlRequestBody.tsx:62]: Based on the spec, `message` is a must https://spec.graphql.org/October2021/#sec-Errors.Error-result-format
But It's hard to tell if every GraphQL impl. follows this rule, can we guard if `message` isn't actually null/empty? so it's not adding the item if there's nothing to be shown.

Comment (malwilley) [reply static/app/components/events/interfaces/request/graphQlRequestBody.tsx]: Guarded against this in [`00eaf3b` (#51846)](https://github.com/getsentry/sentry/pull/51846/commits/00eaf3b56c495fa100281c8f024af7c073b21751)

**Comment by malwilley** [comment]
@marandaneto added line/col numbers to each error message so there is even more context!

**Review by marandaneto — APPROVED** [review]

**Pull request merged**